Valeriia/Movies_database_project/src/pages/MovieDetails/MovieDetails.tsx

```diff
@@ -14,15 +16,34 @@ interface IRouteInfo {
 type MovieProps = MovieConnectProps & RouteComponentProps<IRouteInfo>;
 
 class MovieDetails extends Component<MovieProps, IMovieDetailsState> {
+  fetchData = (offset: number = 0, isLazyLoading: boolean = false) => {
```

Comment:
**Doesn't correspond to the task !!!** 
on MovieDetails Page you don't have search params to make request. You have to fetch movies using movie genre and some default sort params.